core/os/os.cpp

```diff
@@ -591,15 +591,20 @@ bool OS::has_feature(const String &p_feature) {
 	}
 #endif
 
-#ifdef THREADS_ENABLED
 	if (p_feature == "threads") {
+#ifdef THREADS_ENABLED
 		return true;
-	}
 #else
```

Comment:
Is this really needed? `return false` is the default for this method if no feature is a match.

Reply:
Yes, as it will make a call on the JS side to query for features, and that call is not free. If we know the answer, better return it right away.